app/src/test/java/com/example/benchproject/data/movie/details/entity/GenresRemoteTest.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.benchproject.data.movie.details.entity
+
+import com.example.benchproject.domain.movie.details.entity.GenresModel
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Test
+
+internal class GenresRemoteTest {
+
+    @Test
+    fun `when toDomain called on GenresRemote should return GenresModel`() {
+        val genresRemote = listOf(GenresRemote(1, "Test1"), GenresRemote(2, "Test2"))
+        val genresModel = listOf(GenresModel(1, "Test1"), GenresModel(2, "Test2"))
+
+        genresRemote.toDomain() shouldBeEqualTo genresModel
```

Comment:
You can inline `generesModel` to avoid redundant variables
```
genresRemote.toDomain() shouldBeEqualTo listOf(...)
```
This applies for other tests as well

Reply:
Ok, I changed it in `GenresRemoteTest` and `GenresModelTest`

Reply:
Maybe if this change is possible in another file I could make PR for these changes